LICENSE.md

```diff
@@ -0,0 +1,5 @@
+Copyright 2018 Joshua Comeau
```

Comment:
I've seen facebook have the year as "2018-present" [example](https://github.com/facebook/react/blob/master/LICENSE#L3). Seems convenient to avoid having to update it every year.

Reply:
Ah, good catch.